mteb/results/model_result.py

```diff
@@ -43,6 +43,12 @@ def _aggregate_and_pivot(
     elif aggregation_level == "task":
         index_columns = ["task_name"]
 
+    elif aggregation_level == "language":
+        index_columns = ["language"]
+        df = df.explode("language").reset_index(
```

Comment:
Hmm, but this treats the score of the task as the score for the language, which is not correct.

A simple example is Miracl:
https://github.com/embeddings-benchmark/results/blob/main/results/GritLM__GritLM-7B/13f00a0e36500c80ce12870ea513846a066004af/MIRACLRetrievalHardNegatives.json

Which has the main score `0.58448` for `ara-Arab` but `0.60572` for `ben-Beng`. Right now I believe you essentially average all of these (the main score for the task) and split it out.